static/javascript/launch.js

```diff
@@ -528,10 +526,11 @@ function loadSupplementaryDataInfo() {
     "sds_dataset_version",
   ];
 
-  const sdsMetadataSection = document.querySelector("#supplementary_data");
+  const sdsMetadataSection = document.querySelector("#sds_data");
   const sdsMetadataField = (key) =>
-    `<div class="ons-field ons-field--inline">${getLabelFor(key)}${getInputField(key, "text", selectedDataset[key], true)}</div>`;
+    `<div class="ons-field ons-field--inline sds_data">${getLabelFor(key)}${getInputField(key, "text", selectedDataset[key], true)}</div>`;
```

Comment:
I don't think we should be adding this `sds_data` class here to define if this is SDS data. We could use a data attribute or something instead